Exiled.Events/EventArgs/Player/CancelledItemUseEventArgs.cs

```diff
@@ -20,7 +20,7 @@ public class CancelledItemUseEventArgs : IPlayerEvent, IUsableEvent
         /// <summary>
         ///     Initializes a new instance of the <see cref="CancelledItemUseEventArgs" /> class.
         /// </summary>
-        /// <param name="player">The player who's stopping the use of an item.</param>
+        /// <param name="player"> The player who's stopping the use of an item.</param>
```

Comment:
why

Reply:
Removed the space, which was something I missed as I had something else and was rightfully told that's not how parameters works.

Reply:
:+1: